source/Octopus.Server.Client/Exceptions/OctopusExceptionFactory.cs

```diff
@@ -51,74 +51,96 @@ public static async Task<OctopusException> CreateException(HttpResponseMessage r
 
         public static OctopusException CreateException(int statusCode, string body)
         {
-            if (statusCode == 400 || statusCode == 409) // Bad request: usually validation error 
+            if (statusCode is 400 or 409) // Bad request: usually validation error 
             {
-                var errors = JsonConvert.DeserializeObject<OctopusErrorsContract>(body);
-                return new OctopusValidationException(statusCode, errors.ErrorMessage, errors.Errors, errors.Details)
```

Comment:
- `errors` can be null here
- OctopusValidationException will throw null exception when `errors.ErrorMessage` or `errors.Errors` is null